test/dialogflow/directives.spec.ts

```diff
@@ -1047,15 +1047,24 @@ describe("DialogFlow Directives", () => {
 
   describe("FacebookAccountLink", () => {
     it("should add a facebook account link card", async () => {
-      app.onIntent("LaunchIntent", {
-        facebookAccountLink: "https://www.messenger.com",
-        flow: "yield",
-        sayp: "Say!",
-        textp: "Text!",
-        to: "entry",
+      app.onIntent("LaunchIntent", (voxaEvent: DialogFlowEvent) => {
+        voxaEvent.model.userId = voxaEvent.user.userId;
+
+        return {
+          facebookAccountLink: "https://www.messenger.com",
+          flow: "yield",
+          sayp: "Say!",
+          textp: "Text!",
+          to: "entry",
+        };
       });
 
+      event = _.cloneDeep(require("../requests/dialogflow/facebookLaunchIntent.json"));
+
       const reply = await dialogFlowAgent.execute(event);
+      const attributesJson = JSON.parse(reply.outputContexts[0].parameters.attributes);
+
+      expect(attributesJson.model.userId).to.equal("1234567890");
```

Comment:
this is a completely different test, please create it separately, you're testing wether we're adding the user id, but the test says 'should add a facebook account link card'